CONTRIBUTING.md

```diff
@@ -166,6 +166,7 @@ eclipse.exe file.
 
 ### Set up a Tomcat server in Eclipse
 
+- If you want to test the Frank!Doc webapplication, you need file `iaf/target/ibis-adapterframework-frankdoc-7.7-SNAPSHOT-frankdoc.jar` on the classpath (or a later version, if you are reading this after the 7.7 release). Right-click the ibis-adapterframework-webapp project and select "Properties". In the dialog that opens, select "Deployment Assembly". To the right, press "Add...". Another dialog opens. Select "Archives from Workspace". Press "Next" and then "Add...". Select the mentioned file by expanding the "iaf" folder in the tree view.
```

Comment:
Is het niet voldoende om de frankdoc.jar als dependency in de pom van de webapp op te nemen?

Reply:
Ik denk dat het frankDoc Maven artifact alleen de doclet bevat, niet de files die je genereert als je het doclet toepast op de F!F code. In ieder geval zag ik vanochtend een lege Frank!Doc website, dus zonder groepen. Pas toen ik de Deployment Assembly aanpaste zoals ik hier beschrijf, werd de website gevuld.

Reply:
Ik heb in mijn .m2 directory een find gedaan op file ibis-adapterframework-frankdoc-7.7-SNAPSHOT-frankdoc.jar. Daar kwam niets uit. Er is dus geen artifact dat een file bevat met deze naam.

Reply:
Ik denk dat het beter is om een build te doen van het IAF. Dan worden namelijk de bestanden gegenereerd, en in de core.jar gestopt. Dan is de stap om handmatig iets aan de deployment assembly toevoegen niet meer nodig.

Zolang de frank-doc-plugin nog niet officiel gereleased is, heb je dan nog wel een extra (plugin) repo nodig om deze te kunnen downloaden uit onze nexus.